tools/shared.py

```diff
@@ -884,9 +891,6 @@ def apply_configuration():
 apply_configuration()
 
 # EM_CONFIG stuff
-if JS_ENGINES is None:
-  JS_ENGINES = [NODE_JS]
```

Comment:
You don't like defaulting to running the tests in NODE_JS?

These two lines allow normal users to avoid having JS_ENGINES in their config files, and still be able to run tests.

Reply:
Oh, I think this is just dead code. Earlier in the file we have
```
JS_ENGINES = []
[...]
JS_ENGINES = [listify(engine) for engine in JS_ENGINES]
```
so it could never be None AFAICT.